cityofaustin/atd-moped
# Add secondary project name to project summary

## Associated issues

This PR intends to close https://github.com/cityofaustin/atd-data-tech/issues/14558.

## Testing

Local testing for this one; there is a new field added to the projects table.

### Testing Process

* Fire up the database, optionally grabbing a production set of data
* Apply migrations and metadata
* Visit any project and edit the name:
  * By clicking directly on the name
  * By choosing "Rename" in the hamburger on the right
* Change a primary project name
  * See that you can not set that primary name to the empty string 
* Define a secondary name
  * See how that is rendered, and note that containing tags are not rendered when it's undefined (the default)
* Blank out a secondary name
* Mess with the viewport to see how things reflow
  * In view mode
  * In edit mode
* Observe the `Snackbar`, do you like the wording? 
---
#### Ship list
